extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -80,9 +82,44 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 		return '';
 	}
 
+	function getBidderWon(slotParams, realSlotPrices) {
+		var slotPricesKeys = Object.keys(realSlotPrices).filter(function(key) {
```

Comment:
Silly question: can't we use here `realSlotPrices.filter(function(value){})` instead of `Object.keys(realSlotPrices).filter(function(key){})`? Is it because `realSlotPrices` may not be an array?